.github/workflows/ruff-format.yml

```
@@ -0,0 +1,10 @@
+name: Ruff
+on: [push, pull_request]
```

only one small comment. I think we will only need on `pull_request`. On `push` is perhaps a bit too much
```suggestion
on: [pull_request]
```

---

ok